server/runtime/src/test/java/org/infinispan/server/resilience/GracefulShutdownRestartIT.java

```diff
@@ -0,0 +1,58 @@
+package org.infinispan.server.resilience;
+
+import static org.infinispan.server.security.Common.sync;
+import static org.junit.Assert.assertEquals;
+
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.commons.util.Eventually;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.server.test.ContainerInfinispanServerDriver;
+import org.infinispan.server.test.InfinispanServerRule;
+import org.infinispan.server.test.InfinispanServerTestConfiguration;
+import org.infinispan.server.test.InfinispanServerTestMethodRule;
+import org.infinispan.server.test.ServerRunMode;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @since 10.0
+ */
+public class GracefulShutdownRestartIT {
```

Comment:
isn't missing tests for `stop` and `stop-with-servers`?

Reply:
That's not a graceful shutdown. And  the REST endpoint has a test for individual server already